shell/app-shell/elements/persistent-arc.js

```diff
@@ -52,17 +52,21 @@ class PersistentArc extends Xen.Debug(Xen.Base, log) {
         const externalManifest = this._getExternalManifest();
         if (externalManifest != metadata.externalManifest) {
           metadata.externalManifest = externalManifest;
-          this._fire('metadata', metadata);
           state.metadata = null;
+          //this._fire('metadata', metadata);
```

Comment:
We really don't want to do this any more? Also why don't we fire metadata for the conditional below at line 60? Maybe because we rely on knowing the firebase update will trigger something subsequently?

This is all probably correct as-is, I'm just checking since it's unfamiliar and figure better to hear brief explanation than spelunk in depth myself.